src/jdk.compiler/share/classes/com/sun/tools/javac/util/Names.java

```diff
@@ -25,6 +25,8 @@
 
 package com.sun.tools.javac.util;
 
+import com.sun.tools.javac.parser.Tokens.TokenKind;
+
```

Comment:
I realize why you need this, but this is a questionable import from an architectural point of view.   Spaghetti imports can be a problem, and the intent has been that `utils` is a collection of low-level utility classes, and the parser is a layer built on top of that:  here, you are adding a reverse dependence, so that a class in `utils` depends on a class in `parser`.